web/public/locales/pl.json

```
@@ -126,13 +126,15 @@
   "misc": {
     "maintitle": "Emisja CO₂ konsumpcji energii elektrycznej w czasie rzeczywistym",
     "oops": "Ups! Mamy problemy z komunikacją z serwerem. Spróbujemy ponownie za kilka sekund.",
-    "newversion": "Jest dostępna nowa wersja! <a onClick=\"location.reload(true);\">Załaduj ją</a>.",
+    "newversion": "Jest dostępna nowa wersja!",
     "retrynow": "Spróbuj teraz",
     "database-ad": "Szukasz danych historycznych? <a href=\"https://electricitymaps.com?utm_source=app.electricitymaps.com&utm_medium=referral\" target=\"_blank\">Sprawdź Electricity Map Database!</a>",
     "api-ad": "Chcesz mieć aktualne dane w twojej aplikacji lub na urządzeniu? <a href=\"https://electricitymaps.com?utm_source=app.electricitymaps.com&utm_medium=referral\" target=\"_blank\">Spróbuj Electricity Map API!</a>",
     "legend": "Legenda",
     "faq": "Najczęściej Zadawane Pytania",
-    "webgl-not-supported": "Nie można wyrenderować mapy, ponieważ wykorzystywana przeglądarka nie obsługuje WebGL."
+    "webgl-not-supported": "Nie można wyrenderować mapy, ponieważ wykorzystywana przeglądarka nie obsługuje WebGL.",
+    "dismiss": "Odrzucać",
+    "reload": "Przeładować"
```

In Polish, "odrzuć" is "dismiss" in a stronger sense of "reject", and I think equivalent of "ignore" works better. "Przeładować" is something you do with cargo at a port, for reloading web pages we use "odśwież". Also both of these need to be in imperative voice.

```suggestion
    "dismiss": "Ignoruj",
    "reload": "Odśwież"
```